C++14/dp/lcs.cpp

```diff
@@ -0,0 +1,30 @@
+
+
+// Length of Longest Inceasing Subsequence
+// DYNAMIC PROGRAMMING APPROACH - O(m*n) where m and n are the length of strings
+
+
+
+int lcs(string s, string t) {
+	int m = (int)s.length();
+	int n = (int)t.length();
+
+
+	vector<vector<int> > dp(m+1, vector<int>(n+1));
+
+	for(int i=0; i<=m; i++) {
+		for(int j=0; j<=n; j++) {			// dp[i][j] stores the length of lcs of {s[0:i-1] and t[0:j-1]}
```

Comment:
Move comments to a separate line.

Comment on line 16 before for loop.
Comment on other lines before the if/else conditions.